routes/helpers/localize_results.js

```diff
@@ -0,0 +1,37 @@
+const localize =  (locale, dbResultsObj) => {
+
+  // keynames in DB that contain translatable content follow pattern 'name___locale'
+  const separator = '___';
+  const translatableKeyReg = /\_{3}.+$/;
+
+  // gets array of translatable values from DB results
+  const translatableFields = Object.keys(dbResultsObj)
+    .filter(key => translatableKeyReg.test(key))
+    .map(filteredKey => filteredKey.replace(translatableKeyReg, ''));
+
+  // create new results object with relevant data only
+  return Object.keys(dbResultsObj)
+    .filter(key => {
+      const [ type, loc, ] = key.split(separator);
+      return translatableFields.indexOf(type) < 0 || loc === locale;
```

Comment:
Why are there 2 conditions here?

What we are saying:

  1. Does the field have a translation? OR
  2. is the field the translation we are looking for?

I think the first condition is unnecessary.

Ie if 2 is true, is there a situation where 1 is false?